routers/web/web.go

```diff
@@ -484,6 +484,10 @@ func registerRoutes(m *web.Route) {
 	m.Group("/user/settings", func() {
 		m.Get("", user_setting.Profile)
 		m.Post("", web.Bind(forms.UpdateProfileForm{}), user_setting.ProfilePost)
+		m.Group("/account", func() {
+			m.Combo("").Get(user_setting.Account).Post(web.Bind(forms.ChangePasswordForm{}), user_setting.AccountPost)
```

Comment:
Postの方は消しておいたままでいいと思います

```suggestion
			m.Get("", user_setting.Account)
```